### DISCUSSION TITLE
does approve() method update the data ?

### DISCUSSION THREAD

**Discussion by thisnugroho** [cjmellor/approval#12] [Q&A]
Hello, i have a question when i run this `$record->approve()` does it update the data or just change the state value in table approvals ?

thank you.

**Comment by cjmellor** [comment]
Hi,

It will update the `state` column to `approved`

**Reply by thisnugroho** [reply]
Thank you for the answer 😄,

One more question,
Does it possible to update the data when the state is changed to `approved` or when `$record->approve()` triggered ?
if it's possible how to implement it ?

Many Thanks

**Reply by cjmellor** [reply]
Approving the Model makes it public
